docs/guides/postinstall.md

```diff
@@ -219,14 +219,14 @@ The script below includes all cases with commented sections. Uncomment the relev
        pre/*)
          # Required for all T2 models
          rmmod -f apple_bce
-         
-         # Uncomment the following for models with touchbar but not using tiny dfr
-         #rmmod -f appletbdrm
-         #rmmod -f hid_appletb_kbd
-         #rmmod -f hid_appletb_bl
-         
+
          # Uncomment the following if using tiny dfr for touchbar
          #/etc/init.d/tiny-dfr stop
+         #rmmod -r appletbdrm
```

Comment:
`modprobe -r` not `rmmod -r`

And please see the previous review comments. Some have not been addressed yet.